xbmc/FileItem.cpp

```diff
@@ -733,8 +733,15 @@ bool CFileItem::IsVideo() const
   if (HasPictureInfoTag())
     return false;
 
-  if (IsPVRRecording())
-    return true;
+  if (IsPVR())
```

Comment:
No, the actual problem is that some lines below the 'video' filename url extensions are checked and .pvr (which matches both channels and recordings) and this leads to a false positive. Thus, pvr must be handled without filename extension check.